packages/stark-ui/src/modules/table/components/table.component.html

```diff
@@ -53,7 +52,14 @@
 					</button>
 				</div>
 			</mat-menu>
-		</div>
+		</ng-container>
+		<stark-minimap
+			*ngIf="minimap !== false"
+			[mode]="minimap === 'compact' ? 'compact' : undefined"
+			[items]="_minimapItemProperties"
+			[visibleItems]="_visibleMinimapItems"
+			(showHideItem)="toggleColumnVisibility($event)"
+		></stark-minimap>
```

Comment:
Could you add a tooltip for this minimap? Previously this was shown as tooltip: `STARK.TABLE.TOGGLE_COLUMNS` although it was not correctly implemented because it was always shown even if the Minimap was used alone :s